src/explore-education-statistics-admin/src/pages/publication/PublicationMethodologyPage.tsx

```diff
@@ -0,0 +1,477 @@
+                            onClick={() => {
+                              setDropMethodologyId(methodology.methodologyId);
+                            }}
+                          >
+                            Remove
+                            <VisuallyHidden>
+                              {` ${methodology.title}`}
+                            </VisuallyHidden>
+                          </ButtonText>
+                        )}
+                      </td>
+                    </tr>
+                  );
+                },
+              )}
+              {externalMethodology && (
+                <tr>
+                  <td>{externalMethodology.title}</td>
+                  <td>External</td>
+                  <td />
+                  <td />
+                  <td>
+                    {permissions.canManageExternalMethodology && (
+                      <Link
+                        className="govuk-!-margin-right-4"
+                        to={generatePath<PublicationRouteParams>(
+                          externalMethodologyEditRoute.path,
+                          {
+                            publicationId,
+                          },
+                        )}
+                        unvisited
+                      >
+                        Edit
+                        <VisuallyHidden>
+                          {` ${externalMethodology.title}`}
+                        </VisuallyHidden>
+                      </Link>
+                    )}
+
+                    <Link
+                      className="govuk-!-margin-right-4"
+                      to={externalMethodology.url}
+                      unvisited
+                    >
+                      View
+                      <VisuallyHidden>
+                        {` ${externalMethodology.title}`}
+                      </VisuallyHidden>
+                    </Link>
+
+                    {permissions.canManageExternalMethodology && (
+                      <ButtonText
+                        variant="warning"
+                        onClick={toggleRemovingExternalMethodology.on}
+                      >
+                        Delete
+                        <VisuallyHidden>
+                          {` ${externalMethodology.title}`}
+                        </VisuallyHidden>
+                      </ButtonText>
+                    )}
+                  </td>
+                </tr>
+              )}
+            </tbody>
+          </table>
+        </>
+      ) : (
+        <p>There are no methodologies for this publication yet.</p>
+      )}
+
+      <h3>Other options</h3>
+
+      {permissions.canManageExternalMethodology && !externalMethodology && (
+        <>
+          <Link
+            to={generatePath<PublicationRouteParams>(
+              externalMethodologyEditRoute.path,
+              {
+                publicationId,
+              },
+            )}
+          >
+            Add external methodology
```

Comment:
Instead of doing this I've made it so add/edit external methodology and adopt a methodology are sub pages within the manage publication tabs instead of linking out to completely separate pages.

When all this work is ready to go live and I do a massive tidy up I'll probably move the form components for these into the publication/components folder, but for now have left them with the pages in the methodology folder.